exporter.py

```diff
@@ -31,7 +31,10 @@ def xanes_textout(
     """
 
     h = tiled_client_raw[scanid]
-    filepath = f"/nsls2/data/srx/proposals/{h.start['cycle']}/{h.start['data_session']}/scan_{h.start['scan_id']}_xanes.txt"  # noqa: E501
+    if "SRX Beamline Commissioning".lower() in h.start["proposal"]["proposal_title"].lower():
+        filepath = f"/nsls2/data/srx/proposals/commissioning/{h.start['data_session']}/scan_{h.start['scan_id'    ]}_xanes.txt"
+    else:
+        filepath = f"/nsls2/data/srx/proposals/{h.start['cycle']}/{h.start['data_session']}/scan_{h.start['sca    n_id']}_xanes.txt"  # noqa: E501
```

Comment:
Not sure if it's showing up on your end, but looks like there are extra spaces on lines 35 and 37:
`h.start['scan_id'    ]` and `h.start['sca    n_id']`

Reply:
Extra spaces removed. Artifact of copy-paste